api/src/data_migration/transformation/transform_oracle_data_task.py

```diff
@@ -49,6 +50,7 @@ class TransformOracleDataTaskConfig(PydanticBaseEnvConfig):
     enable_opportunity_attachment: bool = (
         True  # TRANSFORM_ORACLE_DATA_ENABLE_OPPORTUNITY_ATTACHMENT
     )
+    enable_competition: bool = True  # TRANSFORM_ORACLE_DATA_ENABLE_COMPETITION
```

Comment:
Let's disable this by default, and manually test it after we merge it. The job will probably take a while to run and we don't want two hourly jobs running on top of each other.